pkg/components/linkerd/manifest.go

```diff
@@ -27,4 +36,59 @@ identity:
 {{ .Cert }}
       keyPEM: |
 {{ .Key }}
+
+# Following is from values-ha.yaml which contains the values needed to enable HA mode.
+enablePodAntiAffinity: true
+
+# controller configuration
+controllerReplicas: 3
+controllerResources: &controller_resources
```

Comment:
Using YAML anchor is not very common. I feel that it makes the configuration less readable. Also, does really all linkerd components use the same amount of resources? I really have doubts about that.

In any case, I'd at least add some comments where the anchor is created and in one place where is used, so one does not need to look up anchors syntax.

Reply:
Still relevant IMO.